x/auth/ante/validator_tx_fee.go

```diff
@@ -39,8 +39,10 @@ func CheckTxFeeWithValidatorMinGasPrices(ctx sdk.Context, tx sdk.Tx) (sdk.Coins,
 			}
 		}
 	}
-
-	priority := getTxPriority(feeCoins, int64(gas))
+	priority := int64(0)
```

Comment:
is 0 the lowest priority? might be useful to add a comment here for future readers

Reply:
yeah it is, I'll leave a comment. Realistically, if a tx has specified 0 gas, it will most likely run out of gas anyways